BookReaderDemo/demo-vendor-fullscreen.html

```diff
@@ -31,7 +31,7 @@
     <script src="../BookReader/plugins/plugin.url.js"></script>
 
     <!-- vendor fullscreen plugin -->
-    <script src="../BookReader/plugins/plugin.vendor-fullscreen.js"></script>
+    <script src="../src/js/plugins/plugin.vendor-fullscreen.js"></script>
```

Comment:
These should be using the build js files, not the src.

```suggestion
    <script src="../BookReader/plugins/plugin.vendor-fullscreen.js"></script>
```

Reply:
Oh sorry, was using for testing purpose. will change this.